viewer/src/dataModels/cad/internal/CadMetadataParserV8.ts

```diff
@@ -39,12 +39,12 @@ export interface CadSectorMetadataV8 {
       xz: number;
     };
     readonly recursiveCoverageFactors:
-    | {
-      xy: number;
-      yz: number;
-      xz: number;
-    }
-    | undefined;
+      | {
```

Comment:
We need to check our lint settings or something, it seems to save differently across machines.